src/config.js

```diff
@@ -16,6 +16,6 @@
 
 export default {
   getAbConfigEndpoint: (example) => {
-    return `https://gist.githubusercontent.com/alexnj/4c8d9198d16b238e4c7040250f052284/raw/${example}`;
+    return `https://gist.githubusercontent.com/alexnj/56e6a1bed280d3111e623f7f6da4d139/raw/${example}`;
```

Comment:
I left a comment on the new version: https://gist.github.com/alexnj/56e6a1bed280d3111e623f7f6da4d139?permalink_comment_id=4349894#gistcomment-4349894